osclib/stagingapi.py

```diff
@@ -110,6 +102,20 @@ def __getattr__(self, attr):
         # Raise AttributeError like normal.
         return self.__getattribute__(attr)
 
+    @property
+    def rings(self):
+        if self._rings:
+            return self._rings
+
+        # If the project support rings, inititialize some variables.
+        if self.crings:
+            self._rings = (
+                '{}:0-Bootstrap'.format(self.crings),
+                '{}:1-MinimalX'.format(self.crings)
+            )
+        else:
+            self._rings = []
+
```

Comment:
initialing `rings` is no longer happened in `__init__`, I thought first time calling `self.rings` has nothing returned but second time does?

Reply:
so should be

```
if self._rings is None:
  initialize it
  
return self._rings
```

Reply:
Did I mention it's just too warm ? :(